lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -151,5 +171,27 @@
         border: 0;
       }
     }
+    .exam-timer-clock {
+      display: inline-block;
+      padding: 0 0 0 ($baseline/3);
+      background-color: $gray-l3;
```

Comment:
@stvstnfrd due to theming updates that have happened on our end, using the `$gray-l3` coloring is no longer integrated into this file.  Could you update this line to `theme-color("light")`, and attach updated screenshots of what this will look like with the theme updates?  Thanks!

Reply:
@fysheets since this PR is a re-open by George, we don't have push access to this branch. I think it's going to be easiest if you guys make the change at this point.